### PR TITLE
Cppclassroom

### PR DESCRIPTION
**Pull request by smallst**
re pr for clean commit

### CODE REVIEW

**Comment by smallst** [comment]
@AndrewJakubowicz prev pr merged with all commits. Wjllance says it is better to squash and mege
if the testing file is not test,(while i wonder you have done it before :joy: )you can test here.

**Comment by AndrewJakubowicz** [comment]
@smallst That makes sense! Squash and merge is good.

I don't see the tests in this diff. I don't think this PR is the same as what was merged and reverted previously. Could you add the tests back or am I missing something?

File: app/core/utils.coffee [hunk 1]
```diff
@@ -1,5 +1,77 @@
 slugify = _.str?.slugify ? _.string?.slugify # TODO: why _.string on client and _.str on server?
 
+translatejs2cpp = (jsCode, fullCode=true) ->
```
Comment (AndrewJakubowicz) [inline app/core/utils.coffee]: This function is missing tests in this PR.

Comment (smallst) [reply app/core/utils.coffee]: oh, my fault! i add /app directory into git repo and missing /test directory :joy:, updated

**Comment by AndrewJakubowicz** [comment]
@smallst Thanks for adding the tests back! Could anything else be different or is everything else the same?

**Comment by wjllance** [comment]
squash merge version for https://github.com/codecombat/codecombat/pull/5729

**Pull request merged**